src/node-router.js

```diff
@@ -9,90 +9,31 @@
 "use strict";
 
 var fluid = require("infusion"),
-    flock = fluid.require("flocking"),
-    minimist = require("minimist");
+    flock = fluid.require("flocking");
 
 fluid.require("%flocking-midi-router/src/midi-router.js");
 
-fluid.defaults("flock.midi.router.node", {
-    gradeNames: "flock.midi.router",
-
-    inputs: "{argProcessor}.options.inputs",
-    outputs: "{argProcessor}.options.outputs",
+fluid.registerNamespace("flock.midi.router.node");
 
-    components: {
-        argProcessor: {
-            type: "flock.midi.router.argProcessor"
-        }
+flock.midi.router.node.startup = function (that) {
+    if (that.options.listPorts) {
+        flock.midi.logPorts();
+        process.exit(0);
     }
-});
-
-fluid.defaults("flock.midi.router.argProcessor", {
-    gradeNames: "fluid.component",
-
-    args: "@expand:flock.midi.router.argProcessor.parseArgs()",
-
-    inputs: {
-        expander: {
-            funcName: "flock.midi.router.argProcessor.parsePortArg",
-            args: ["{that}.options.args.i"]
-        }
-    },
-
-    outputs: {
-        expander: {
-            funcName: "flock.midi.router.argProcessor.parsePortArg",
-            args: ["{that}.options.args.o"]
-        }
-    },
+};
 
+fluid.defaults("flock.midi.router.node", {
+    gradeNames: ["flock.midi.router"],
+    setLogging: fluid.logLevel.FAIL,
```

Comment:
IMO it should match the fluid method that's used to set logging (as it accepts the same arguments).  I will clean this up to match.